exercises/practice/grade-school/.meta/example.c

```diff
@@ -28,7 +40,9 @@ bool add_student(char *name, uint8_t grade)
 {
    bool added = false;
 
-   if (roster.count < MAX_STUDENTS && strlen(name) < MAX_NAME_LENGTH) {
+   if (is_student_on_roster(name, roster)) {
```

Comment:
Nitpick: how about `student_is_on_roster` instead of `is_student_on_roster`? Then it will read more naturally when used with `if`.